homeassistant/components/weather/buienradar.py

```diff
@@ -0,0 +1,119 @@
+"""
+Support for Buienradar.nl weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/weather.buienradar/
+"""
+import logging
+import asyncio
+from datetime import timedelta
+from homeassistant.components.weather import (
+    WeatherEntity, PLATFORM_SCHEMA)
+from homeassistant.const import \
+    CONF_NAME, TEMP_CELSIUS, CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.helpers import config_validation as cv
+# Reuse data and API logic from the sensor implementation
```

Comment:
It should be the other way around. I guess that this crept in as the weather component is not complete. Anyway, good for now.